docs/structured-streaming-programming-guide.md

```diff
@@ -1225,6 +1225,12 @@ Note that there are some restrictions when you use session window in streaming q
 
 For batch query, global window (only having `session_window` in grouping key) is supported.
 
+Due to implementation details of session window in streaming query, for each input row in same session,
+it will be assigned initial session window and then merged into real session after shuffling. If the RPS
```

Comment:
Shall we use the full word instead of `RPS`?

Reply:
yea, let me revise it. Thanks.